docs/releases/README.md

```diff
@@ -0,0 +1,42 @@
+# Release Workflow for Ega Metadata Schemas
+In this document you will find the workflow for creating releases of the EGA Metadata Schemas project. It provides a step-by-step diagram to help illustrate the steps involved.
+
+## Workflow Diagram
+![Release Workflow Diagram](images/Release_workflow_diagram.svg)
```

Comment:
I would suggest to explicitly mark the start of the flow in the diagram for clarity similar to how you marked the "Process finished". I understand it starts at the top and the flow goes down but since the starting step is similar to the ones next to it would be nice to have it marked.

Reply:
Roger that, good idea